farm/modeling/biadaptive_model.py

```diff
@@ -0,0 +1,537 @@
+import copy
+import json
+import logging
+import os
+from argparse import Namespace
+from pathlib import Path
+
+import multiprocessing
+import numpy
+import torch
+from torch import nn
+
+from farm.data_handler.processor import TextSimilarityProcessor
+from farm.data_handler.data_silo import DataSilo
+from farm.modeling.language_model import LanguageModel
+from farm.modeling.prediction_head import PredictionHead, TextSimilarityHead
+from farm.modeling.tokenization import Tokenizer
+from farm.utils import MLFlowLogger as MlLogger, stack
+
+logger = logging.getLogger(__name__)
+
+
+class BaseBiAdaptiveModel:
+    """
+    Base Class for implementing AdaptiveModel with frameworks like PyTorch and ONNX.
```

Comment:
Change docstring to "Base Class for implementing a BiAdaptiveModel that consists of **two** language models and a prediction head.
For example, such an architecture is used in the Dense Passage Retriever."